Formula/p/python@3.12.rb

```diff
@@ -395,12 +395,23 @@ def post_install
        install.
 
        If you wish to install a non-brew-packaged Python package,
-       create a virtual environment using python3 -m venv path/to/venv.
-       Then use path/to/venv/bin/python and path/to/venv/bin/pip.
+       use a virtual environment:
+
+         python -m venv path/to/venv
+         source env/bin/activate
+         python -m pip install xyz
 
        If you wish to install a non-brew packaged Python application,
        it may be easiest to use pipx install xyz, which will manage a
-       virtual environment for you. Make sure you have pipx installed.
+       virtual environment for you. You can install pipx with
+       brew install pipx.
```

Comment:
```suggestion

         brew install pipx
```

Breaking this out like the other example above seems appropriate. Not strongly for it but it feels better.

Alternatively:
```suggestion
       'brew install pipx'.
```